index.html

```diff
@@ -38,26 +38,13 @@
       <main>
         <h1>Welcome to Malcolm’s CSD120 Website!</h1>
         
-      
-       <div class= "shadow">
-        <picture>
-          
-          <a href="Images/MalPic.jpg"     alt ="Thumbnail">
-          <source media="(min-width: 300px)" srcset="Images/MalPicThumb300wide.jpg x2" alt="Thumbnail" />
-          <source media="(min-width: 150px)" srcset="Images/MalPicThumb150wide.jpg x1" alt = "Thumbnail"/>
-          <img src="Images/MalPicThumb300wide.jpg" alt=" Thumbnail" class= padding/>
-        
-         
-        </picture>
```

Comment:
Still some issues getting previous labs up to snuff based on my pull requests.  Feel free to touch base with me if you want to confirm that you've merged things correctly